src/views/certifications/views/[slug]/server.ts

```diff
@@ -3,13 +3,16 @@ import {
 	getAllCertificationProgramSlugs,
 	getCertificationProgram,
 } from 'views/certifications/content/utils'
+import { ProgramSlug } from 'views/certifications/types'
 import { CertificationProgramViewProps } from './types'
 
 export async function getStaticProps({
 	params: { slug: rawSlug },
 }: GetStaticPropsContext): Promise<{ props: CertificationProgramViewProps }> {
 	// Get the `slug` as a string
-	const slug = Array.isArray(rawSlug) ? rawSlug.join('/') : rawSlug
+	const slug = (
+		Array.isArray(rawSlug) ? rawSlug.join('/') : rawSlug
```

Comment:
Whats the case where this would be an array? I thought only catch-all routes had the potential to be an array?

Reply:
Or is this just a typescript weirdness thing?

Reply:
TypeScript weirdness is a good summary I think 😬 But now I wonder whether there's something I can do with the `GetStaticPropsContext` here to mitigate this, slash signal that we're not in a catch-all route - cause +1 that this shouldn't have the potential to be an array 🤔

Reply:
Yep yep, I can pass `GetStaticPropsContext<{ slug: ProgramSlug }>` here, I guess to kind of encode the combined `getStaticPaths` and `[slug].tsx` page file setup that NextJS would otherwise be unaware of 👍 This feels way cleaner - thanks for flagging 🙇